askama_parser/src/lib.rs

```diff
@@ -54,12 +54,12 @@ mod _parsed {
 pub use _parsed::Parsed;
 
 #[derive(Debug)]
-pub struct Ast<'a> {
+struct Ast<'a> {
```

Comment:
Hmm, I'd prefer to go the other way on this. I think we should add a `pub fn nodes(&self) -> &[Node<'_>]` accessor and keep `Parsed` as an optional addition.